wizeline/project-lab
# Fix - zod schema for repos and issue from join projects modal

#### What does this PR do?

This PR solves 2 issues:
- The Join Project Modal not being submitted after pressing the button
- The repoUrls field from the edit project form blocking the submission

#### Where should the reviewer start?

They can check the Edit project form and the Join Project modal in any order

#### How should this be manually tested?

For the join project modal:
- Go to an existing project where you are not a member of
- Click in Join project
- Fill the form and click on the Join Project button
- You should see that the modal is hidden and now you are part of the project

For the repoUrls field:
- Go to a project that has at least one value in the repoUrl and click on edit
- Add and/or delete the info fro this field
- Click on the submit button at the bottom of the page
- The project should now display the updated info for this field without any issues.

#### Checklist




# UPDATE

- I just pushed some changes that fix the issue regarding the searchProjects endpoint being called twice when loading the homepage. Now it only calls this endpoint once with the required info.
